scipy/stats/_continued_fraction.py

```diff
@@ -291,8 +293,15 @@ def func(n, *args):
     # on each callable to get the shape and dtype, then we broadcast these
     # shapes, compute the result dtype, and broadcast/promote the zeroth terms
     # and `*args` to this shape/dtype.
+
     # `float32` here avoids influencing precision of resulting float type
-    zero = xp.asarray(0, dtype=xp.float32)
+    # patch up promotion: in numpy (int64, float32) -> float64, while in torch
```

Comment:
@mdhaber I thought I'd double check whether you were fine with this change

Reply:
I might refactor when I look at this function again, but seems fine.